argo/cron/bank2-compaction.yaml

```
@@ -0,0 +1,50 @@
+# maintainer: cosven
+
+metadata:
+  name: tipocket-bank2-compaction
```

```suggestion
  name: tipocket-bank2-compaction-opt
```
this is testing some compaction related optimization, not compaction itself. Also change the name of bank2-compaction.yaml.